tools/build.sh

```diff
@@ -1,11 +1,33 @@
 #!/bin/bash
 
-# Note: Not sure if we use this. It's not straightforward to install on OSX
-# if ! which envsubst > /dev/null 2>&1; then
-#  echo "You don't seem to have envsubst yet. Go get it! Using sed as an " \
-#    "alternative approach is too slow... :("
-#  exit 1
-# fi
+# execute config file given as parameter; default to testnet
+CONFIG_FILE=testnet
+if [ "$1" != "" ]; then
+    CONFIG_FILE="$1"
+fi
+
+if [ ! -f config/$CONFIG_FILE.conf ]; then
+    echo "Config file not found!"
```

Comment:
Would be nice to see the file path that was looked for but could not be found in the error message as well.